dts/arm/nordic/nrf52832.dtsi

```diff
@@ -87,6 +89,7 @@
 			 *              "nordic,nrf-spim" or
 			 *              "nordic,nrf-spis".
 			 */
+			compatible = "nordic,nrf-spim";
```

Comment:
Are you sure? What about PAN 58? See CONFIG_SOC_NRF52832_ALLOW_SPIM_DESPITE_PAN_58 for details.

Reply:
Nice catch. I'll change this to `nordic-nrf-spi` for nRF52832.